src/plugins/discover/public/application/components/doc_viewer_links/doc_viewer_links.scss

```diff
@@ -0,0 +1,21 @@
+.osdDocViewerLinks {
+  .euiListGroup {
```

Comment:
I think we need to use out of the box EUI classes/components for this instead of writing custom CSS so that we can easily maintain this.

Reply:
Hey @tmarkley, thanks.
Changed to EUI Flex components instead of using custom CSS.
Also changed the text size to the desired one:

<img width="1403" alt="Screen Shot 2022-04-20 at 18 37 30" src="https://user-images.githubusercontent.com/12782945/164270068-a70d842b-2144-46b0-b4f1-fa67843e739b.png">

@kavilla FYI